2D Simulation/initial_board.html

```diff
@@ -28,13 +28,25 @@
         var container = document.createElement('div');
 
         container.innerHTML = localStorage.getItem("container");
+
+
+     console.log(container.innerHTML);
         container.style.fontSize = "15px";
         container.style.fontWeight = 'bold';
 
         container.id = 'copy_container';
 
         document.body.appendChild(container);
-        // $('#copy_container').addClass('shrink');
+
+        let finalBlocks = JSON.parse(localStorage.getItem('finalBlocks'));
+        console.log(finalBlocks);
```

Comment:
And this.

Reply:
I will need to make a bunch of changes to make all of this compatible with your file moves, so I can take these out then.

Reply:
Whoops just saw your comment above. The js definitely should not be in the html files so I am all about your changes, I will just change the base branch later later today and actually remove the logs.